cpp/src/io/parquet/page_data.cu

```diff
@@ -1897,6 +1897,15 @@ void PreprocessColumnData(hostdevice_vector<PageInfo>& pages,
       if (out_buf.size == 0) {
         int size = thrust::reduce(rmm::exec_policy(stream), size_input, size_input + pages.size());
 
+        // if we are not doing a trim pass, handle the corner case where we somehow have columns
+        // that contain more rows than are in the row group.  we have seen explicit examples of
+        // files being created this way.
```

Comment:
Recommend to polish the comment a bit :smile:

Reply:
Done.